pkg/storage/stores/shipper/shipper_index_client.go

```diff
@@ -95,6 +95,10 @@ func (i *IndexGatewayClientConfig) RegisterFlagsWithPrefix(prefix string, f *fla
 	f.StringVar(&i.Address, prefix+".server-address", "", "Hostname or IP of the Index Gateway gRPC server running in simple mode.")
 }
 
+func (i *IndexGatewayClientConfig) RegisterFlags(f *flag.FlagSet) {
```

Comment:
I can't find usage of this, can you please help me understand why it was added?

Reply:
If we don't implement a `RegisterFlags`, `IndexGatewayClientConfig` won't have support for default flags, which is used by our tests under `stores/shipper`. More specifically, the following block of code is responsible for requiring this:
```go
# pkg/storage/stores/shipper/gateway_client_test.go
	var cfg IndexGatewayClientConfig
	cfg.Mode = indexgateway.SimpleMode
	flagext.DefaultValues(&cfg)
	cfg.Address = storeAddress
```